client/src/data/locales/es-ES.json

```diff
@@ -1,6 +1,47 @@
+  "home.quote": "...trabajando en crear un ecosistema en restauración más equitativo para todos...",
+  "home.extra-work.h2": "¡Echa un vistazo a nuestros otros proyectos!",
+  "caseStudy.title": "¿Por qué es importante esto?",
+  "caseStudy.par1": "Obtener una Licencia para la Venta de Bebidas Alcohólicas, durante mucho tiempo, ha significado formar parte de un \"club exclusivo\" de propietarios de restaurantes, empresas de hostelería y abogados que han dominado el proceso. Históricamente, esto ha dificultado que gente nueva en el sector pueda obtener estas licencias y creado una escasez artificial que ha creado sequías en muchos barrios de la ciudad. A menudo en barrios con menos tradición blanca y con un carácter más diverso.",
+  "caseStudy.par2": "Recientemente se han hecho cambios en el sistema de licencias de Boston que mejoran el acceso a las licencias. Una nueva ley aprobada en 2024 busca otorgar licencias a partes específicas de la ciudad y permitir a la gente de la zona abrir nuevos negocios. Pero esto es solo el principio. Encontrar licencias disponibles aún es difícil, y a menudo fuera del alcance para algunos miembros de la comunidad de Boston.",
+  "caseStudy.par3": "Con este sitio estamos trabajando en crear un ecosistema en restauración más equitativo para todos, y permitirte buscar y entender qué licencias hay disponibles para ti.",
+  "caseStudy.image.alt": "Cuadrado blanco vacío de tamaño medio",
+  "caseStudy.button.read": "LEE EL CASO PRÁCTICO COMPLETO",
+  "extra-work.offsite": "Logo de Off Site, un logo simple en una tipografía sans serif en negrita",
+  "extra-work.bar-management-bootcamp": "Varios estudiantes sentados en una barra escuchando al presentador del bootcamp",
+  "extra-work.code-for-boston": "Logo de Code for Boston, un quinqué de líneas simples con un paréntesis angular en lugar de la llama, con \"Code for Boston\" escrito al lado derecho",
+  "ui.backToHomeButton": "VOLVER AL INICIO",
+  "footer.disclaimerText": "La información proporcionada en esta página web es sólo para fines de información general. No se garantiza que los datos presentados en este sitio sean exactos ni completos. La información está sujeta a cambios sin aviso previo.",
+  "footer.builtBy": "Construido por Code for Boston para Offsite Hospitality",
+  "footer.copyright": "©, OFFSITE LLC Todos los derechos reservados",
+  "footer.siteMap": "Mapa del sitio",
+  "footer.homepage": "Inicio",
+  "footer.map": "Mapa",
+  "footer.database": "Base de Datos",
+  "footer.resources": "Important Resources",
+  "footer.importantLinks": "Enlaces Importantes",
+  "footer.bostonBoard": "Boston Licensing Board",
+  "footer.abcc": "Alcoholic Beverages Control Commission",
+  "footer.application": "Boston License Application",
+  "footer.analyzeBoston": "Analyze Boston",
```

Comment:
Are this proper names? Should they be translated? If they had the links I could look up if they have official translations of their name in their sites.

Reply:
`Important Resources`, `Boston License Application`, `More From Us` should be translated to Spanish

`Boston Licensing Board` `Alcoholic Beverages Control Commission`, and `Analyze Boston`, `Offsite Hospitality`, `Bar Management Bootcamp`, `Code For Boston` should remain in English.

I'll put a note to remove Proper names from the translation system

Reply:
Done